src/lightning/fabric/strategies/fsdp.py

```diff
@@ -49,6 +49,7 @@
     )
 
 _FSDP_ALIASES = ("fsdp", "fsdp_cpu_offload")
+_FSDP_FLATTENED = "_fsdp_flattened"
```

Comment:
What's the benefit of making this a global if it's only used in `_optimizer_has_flat_params`?

Reply:
My thought was that since it was a widely used attribute, it might be worth making accessible to other functions since other functions will likely use it in the future and it would prevent the need to move it. I'm happy to move it to make it function scoped if you think that's better though.

Reply:
I would remove it since it is only used in that one function. But either way is fine with me too

Reply:
Narrowed the ``_fsdp_flattened`` suffix definition to be function scoped as suggested.